docs/node/web3.mdx

```diff
@@ -209,6 +209,68 @@ service] and register it in the service manager for your platform.
 
 :::
 
+### Metrics
+
+Web3 Gateway can report a number of metrics to Prometheus server. By default,
+no metrics are collected and reported. There is one way to enable metrics
+reporting by configuring the `monitoring` section in the config file of your
+Web3 gateway.
+
+```yaml title='gateway.yml'
+...
+# Existing fields omitted ...
+gateway:
+# Existing fields omitted ...
+  monitoring:
+    # Change host to your external IP address if you have users accessing Web3 from the outside.
+    host: "localhost"
+    # Use different port for each Web3 gateway instance, if all run locally.
+    port: 9999
+```
+
+This will run a Prometheus metrics server along with Web3 Gateway.
```

Comment:
I would skip the instructions on configuring the Prometheus part since it is a widely known project with good documentation.